treebrowser/src/treebrowser.c

```diff
@@ -208,7 +206,7 @@ utils_pixbuf_from_stock(const gchar *stock_id)
 static GdkPixbuf *
 utils_pixbuf_from_path(gchar *path)
 {
-#if defined(HAVE_GIO) && GTK_CHECK_VERSION(2, 14, 0)
+#if GTK_CHECK_VERSION(2, 14, 0)
```

Comment:
The GTK check is also mostly irrelevant given how old GTK 2.24 is, and that Geany is depending on 2.24 for a long time now.  But that's outside of the scope of this PR.